doc/io.rst

```diff
@@ -641,9 +641,13 @@ GRIB format via cfgrib
 ----------------------
 
 xarray supports reading GRIB files via ECMWF cfgrib_ python driver and ecCodes_
-C-library, if they are installed. To open GRIB file supply ``engine='cfgrib'``
+C-library, if they are installed. To open a GRIB file supply ``engine='cfgrib'``
 to :py:func:`~xarray.open_dataset`:
 
+.. ipython:: python
+
+    ds_grib = xr.open_dataset('example.grib', engine='cfgrib')
```

Comment:
If `example.grib` doesn't actually exist when we build the docs, this will give a nasty error message. It would be better to use a `:verbatim:` directive here -- scroll open to the opendap examples to see what that looks like.

Reply:
Ouch! I noticed that there was no docs testing in Travis-CI and wondered if you where testing the docs at all. I should have built the docs myself. What is the intended way do you build the docs? `python sertup.py build_sphinx` fails on my setup due to missing `numpydoc`. 

I'm fixing it as you suggest.

Reply:
I added the `:verbatim:` directive, but I was not able to test the build of the documentation due to several errors when building the gallery (a number of core dump in GEOS, maybe due to the setup on my MacOS).

The fix looks trivial enough that it may work.

Reply:
I think this would work:
```
.. ipython::
    :verbatim:

    In [3]: ds_grib = xr.open_dataset('example.grib', engine='cfgrib')
 ```

We actually do test the doc build on Travis-CI, but unfortunately there's no easy way to see generated docs and also errors in `ipython` directive blocks don't stop the build (there's a bug we filed about this somewhere).

Reply:
Fixed. I mis-read the sample.